src/uwtools/tests/test_schemas.py

```diff
@@ -144,6 +237,153 @@ def ww3_prop():
     return partial(schema_validator, "ww3", "properties", "ww3", "properties")
 
 
+# cdeps
```

Comment:
The methods below test various parts of the CDEPS config schema in ways (and with comments) that hopefully illustrate the rules that the schema is trying to enforce. Any of this can be updated, now or later, if the schema needs to be corrected.